src/madetomeasure_api/routes/subscribers.clj

```diff
@@ -1,12 +1,39 @@
 (ns madetomeasure-api.routes.subscribers
   (:require [madetomeasure-api.layout :as layout]
-            [compojure.core :refer [defroutes GET POST DELETE PATCH]]))
+            [madetomeasure-api.db.core :as db]
+            [cheshire.core :refer :all]
+            [compojure.core :refer [defroutes GET POST DELETE PATCH]]
+            [madetomeasure-api.middleware.validate-json :refer [json-schema-validate]])
+  (:import java.sql.SQLException))
+
+(def default-subscriber-map
+  ^{:private true}
+  {:first_name nil :last_name nil})
```

Comment:
Is this like a functional null object pattern or are we required to set up data here?  Ignorant question, sorry.  :information_desk_person: